src/Symfony/Component/Validator/Constraints/Callback.php

```diff
@@ -27,8 +27,8 @@ class Callback extends Constraint
     public $callback;
 
     /**
-     * @param string|string[]|callable|array<string,mixed>|null $callback The callback definition
-     * @param string[]|null                                     $groups
+     * @param callable-string|string[]|callable|array<string,mixed>|null $callback The callback definition
```

Comment:
When a string is passed, it's interpreted as a method name. Which is not a callable-string.

Reply:
Oh yes, got it. Removed from `Choice` as well.